src/modules/CRUD.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+export let tasks = JSON.parse(localStorage.getItem('tasks'));
+export const tasksList = document.querySelector('ul');
+export const addTaskField = document.querySelector('.new-task-field input');
+
+export const removeTask = (index) => {
+  const zeroBasedIndex = index - 1;
+  // remove task by index
+  tasks.splice(zeroBasedIndex, 1);
+  // update indexs
+  for (let i = zeroBasedIndex; i < tasks.length; i++) {
+    tasks[i].index -= 1;
+  }
+};
+
+export const removeAll = () => {
+  tasks = [];
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+export const addTask = (taskDescription) => {
+  const newTask = new taskObject(taskDescription, false, tasks.length + 1);
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure your app is working as a user can't add a task. here's the error from the console below:

![ensure](https://user-images.githubusercontent.com/59397159/174203967-9e68c4ab-5caa-48b2-8260-c93ed9e839ab.png)